web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Gallery/page/ImageForm/ImageDisplay/index.tsx

```diff
@@ -18,6 +18,7 @@ export const ImageDisplay = (props: {
   // Verified images are cached by cloudfront and Next/Image actually causes slower load times.
   if (type === "verified") {
     return (
+      // eslint-disable-next-line @next/next/no-img-element
```

Comment:
@andy-t-wang Please leave a comment why did you disable this
```suggestion
      // eslint-disable-next-line @next/next/no-img-element -- comment here
```